model/src/main/kotlin/config/ErrorResolution.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model.config
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+
+/**
+ * Defines the resolution of an error. This can be used to silence false positives, or errors that have been identified
+ * as not being relevant.
+ */
+data class ErrorResolution(
```

Comment:
Should we more generally call this `IssueResolution` already now, so we could also use it to resolve warnings (once we add them) later?